ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/FieldType.java

```diff
@@ -22,5 +22,6 @@ public enum FieldType {
   Flags,
   FlagLauncher,
   FlagType,
-  FlagDetail
+  FlagDetail,
+  WaysToPay
```

Comment:
I think this is not necessary as we have the other one

Reply:
made the changes. Removed WaysToPay from FieldType